src/Table.js

```diff
@@ -44,17 +47,26 @@ class Table extends React.Component {
 
     const classes = {
       ...getClassSet(bsProps),
-      [prefix(bsProps, 'responsive')]: responsive,
-      [prefix(bsProps, 'inverse')]: inverse,
+      [prefix(bsProps, 'dark')]: dark,
       [prefix(bsProps, 'striped')]: striped,
       [prefix(bsProps, 'bordered')]: bordered,
-      [prefix(bsProps, 'condensed')]: condensed,
+      [prefix(bsProps, 'sm')]: sm,
       [prefix(bsProps, 'hover')]: hover
     };
 
-    return (
+    const table = (
       <table {...elementProps} className={classNames(className, classes)} />
     );
+
+    if (responsive) {
+      let responsiveClass = 'responsive';
+      if (breakpoint) {
+        responsiveClass = `${responsiveClass}-${breakpoint}`;
```

Comment:
What about making `responsive` a prop that can be `true` or a breakpoint? like: `responsive='sm'` or just `responsive`, @taion thoughts?

Reply:
Ah! That would be a better choice in fact.